pkg/spdk/disk/aio/aio.go

```diff
@@ -0,0 +1,78 @@
+package aio
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/pkg/errors"
+
+	spdkclient "github.com/longhorn/go-spdk-helper/pkg/spdk/client"
+	spdktypes "github.com/longhorn/go-spdk-helper/pkg/spdk/types"
+	spdkutil "github.com/longhorn/go-spdk-helper/pkg/util"
+
+	"github.com/longhorn/longhorn-spdk-engine/pkg/spdk/disk"
+)
+
+type DiskDriverAio struct {
+}
+
+const (
+	diskDriver = "aio"
```

Comment:
NIT: Can we use `commonTypes.DiskDriverAio` instead? Same for others.

Reply:
Sure! Addressd, thank you.